openstef/model/regressors/custom_regressor.py

```diff
@@ -28,7 +28,7 @@ def valid_kwargs() -> list[str]:
 
     @classmethod
     @abstractmethod
-    def objective(self) -> Type[RegressorObjective]:
+    def objective() -> Type[RegressorObjective]:
```

Comment:
Should this be staticmethod?
Classmethod takes cls as argument

Reply:
Good catch.